app/scripts/modules/amazon/src/loadBalancer/configure/application/createApplicationLoadBalancer.controller.ts

```diff
@@ -0,0 +1,540 @@
+import { module, IScope, IPromise } from 'angular';
+import { IModalInstanceService } from 'angular-ui-bootstrap';
+import { StateService } from '@uirouter/angularjs';
+import { chain, clone, find, filter, map, trimEnd, uniq, values } from 'lodash';
+
+import {
+  ACCOUNT_SERVICE,
+  AccountService,
+  Application,
+  CACHE_INITIALIZER_SERVICE,
+  CacheInitializerService,
+  IAccount,
+  IGroupsByAccount,
+  INFRASTRUCTURE_CACHE_SERVICE,
+  InfrastructureCacheService,
+  IRegion,
+  IRegionAccount,
+  ISubnet,
+  LOAD_BALANCER_WRITE_SERVICE,
+  LoadBalancerWriter,
+  NAMING_SERVICE,
+  NamingService,
+  SECURITY_GROUP_READER,
+  SecurityGroupReader,
+  SUBNET_READ_SERVICE,
+  SubnetReader,
+  TASK_MONITOR_BUILDER,
+  TaskMonitorBuilder,
+  TaskMonitor,
+  V2_MODAL_WIZARD_SERVICE
+} from '@spinnaker/core';
+
+import { AWSProviderSettings } from 'amazon/aws.settings';
+import { IALBListener,
+  IAmazonApplicationLoadBalancer,
+  IAmazonApplicationLoadBalancerUpsertCommand
```

Comment:
(not suggesting a change in this PR, just something to discuss...)

Should we introduce a "command" or "ops" package to separate domain data and operations?